apps/dashboard/src/components/engine/EnginePageLayout.tsx

```diff
@@ -0,0 +1,270 @@
+import { Spinner } from "@/components/ui/Spinner/Spinner";
+import { useAccount } from "@3rdweb-sdk/react/hooks/useApi";
+import { Flex, Stack } from "@chakra-ui/react";
+import { CircleAlertIcon } from "lucide-react";
+import Link from "next/link";
+import { useCallback, useMemo } from "react";
+import { FiArrowLeft } from "react-icons/fi";
+import invariant from "tiny-invariant";
+import { Heading, Text } from "tw-components";
+import { useDashboardRouter } from "../../@/lib/DashboardRouter";
+import {
+  type EngineInstance,
+  useEngineInstances,
+} from "../../@3rdweb-sdk/react/hooks/useEngine";
+import { SidebarNav } from "../../core-ui/sidebar/nav";
+import type { Route } from "../../core-ui/sidebar/types";
+import { useTrack } from "../../hooks/analytics/useTrack";
+import { EngineVersionBadge } from "./badges/version";
+import { useHasEnginePermission } from "./useHasEnginePermission";
+
+export type ActivePage =
+  | "overview"
```

Comment:
"/overview" is a tab but the link needs to go to `/` (right now link goes to `<x>/overview` which is a 404 because overview is actually at `/index`

Reply:
either change the url or make the index page into -> overview  and redirect `/` -> `/overview` :)

Reply:
Fixed link